js/models/conversations.js

```diff
@@ -663,6 +665,7 @@
     async onFriendRequestTimeout() {
       // Unset the timer
       if (this.unlockTimer) clearTimeout(this.unlockTimer);
+      if (this.isFriend) return;
```

Comment:
```suggestion
      if (this.isFriend()) return;
```

This should be placed after
```js
this.unlockTimer = null;
```

Reply:
Lol I'm dum